src/test/java/org/jenkinsci/plugins/durabletask/PowershellScriptTest.java

```diff
@@ -155,6 +164,7 @@
         c.writeLog(ws, baos);
         assertEquals(0, c.exitStatus(ws, launcher).intValue());
         assertThat(baos.toString(), containsString("Hello, World!"));
+        c.stop(ws,launcher);
```

Comment:
Why would this be necessary? The test failure seemed to indicate a file handle leak as far as I can tell, and the actual `DurableTaskStep` would not normally be calling `stop` here (only if you deliberately abort the build).